massa-models/src/api.rs

```diff
@@ -15,6 +15,7 @@ use massa_time::MassaTime;
 use serde::{Deserialize, Serialize};
 use std::collections::BTreeMap;
 use std::net::IpAddr;
+use strum::Display;
```

Comment:
Why do we need strum here? Cannot we implement Display on our own here?

Reply:
Of course we could implement it.